src-tauri/src/platform_specific/linux.rs

```diff
@@ -1,8 +1,78 @@
 // Linux specific code
 
 use regex::Regex;
+use std::env;
 use std::process::Command;
 
+fn get_proton_dir() -> Option<String> {
+    let steam_dir = steamlocate::SteamDir::locate()?;
+    let compat_dir = format!("{}/compatibilitytools.d/", steam_dir.path.display());
+
+    Some(compat_dir)
+}
+
+/// Downloads and installs NS proton
+/// Assumes Steam install
+pub fn install_ns_proton() -> Result<(), thermite::prelude::ThermiteError> {
+    // Get latest NorthstarProton release
+    let latest = thermite::core::latest_release()?;
+
+    let temp_dir = env::temp_dir();
+    let path = format!("{}/nsproton-{}.zip", temp_dir.display(), latest);
+
+    // Download NorthstarProton release
+    let archive = thermite::core::download_ns_proton(latest, path.clone())?;
+    dbg!("Download done");
+
+    let compat_dir = get_proton_dir().unwrap();
+    std::fs::create_dir_all(compat_dir.clone())?;
+
+    // Extract to Proton dir
+    thermite::core::install_ns_proton(&archive, compat_dir)?;
+    dbg!("Install done");
```

Comment:
flight core has a logging crate 
```suggestion
    log::info!("Install done");
```
iirc